src/go.mod

```diff
@@ -32,6 +32,7 @@ require (
 	github.com/sirupsen/logrus v1.9.3
 	github.com/spf13/cobra v1.8.0
 	github.com/spf13/pflag v1.0.5
+	github.com/stretchr/testify v1.9.0
```

Comment:
I know this library, and I'm not a big fan. With copilot, the error messages are better than the ones shown by test failures from this lib.